### ISSUE TITLE
Feature to disable enter key

### ISSUE THREAD

**Issue by L0ren2** [Julow/Unexpected-Keyboard#131]
I like the overall design very much. What I don't like at all is that the enter key is so close to the backspace key. On Duolingo and on Discord, everytime I want to correct my spelling, I accidentally press the enter key, which causes me to send the first half of my message... This has been cause to lots of failed Duolingo lessons, for which I am a little sad. 
I would love if there was the possibility to remap the enter key to a swipe on the spacebar. That way it would be out of the way when correcting misspelled words. 
Thanks for this great keyboard!

**Comment by Julow** [comment]
When this happen, do you see a "Done" or "Send" key instead of the enter key ?

When the app expects such a key, the keyboard move the Enter key in a corner to give room for this "action" key. Perhaps a solution would be an option to disable this, the Enter key would stay where it is, the "Done" or "Send" key would be in a corner.

The Enter key is supposed to add a new line into the text box but applications might choose to catch it and still send the message.

**Comment by L0ren2** [comment]
No, there is no such key. Its just the normal "enter" key, but as you 
mentioned, the applications choose to send the message instead.
I have not experienced that any app would move the enter key somewhere 
else, it is always in its usual location.

~Greetings

Am Sat, 07 May 2022 10:24:51 -0700
  schrieb Jules Aguillon ***@***.***>:
> When this happen, do you see a "Done" or "Send" key instead of the 
>enter key ?
> 
> When the app expects such a key, the keyboard move the Enter key in 
>a corner to give room for this "action" key. Perhaps a solution would 
>be an option to disable this, the Enter key would stay where it is, 
>the "Done" or "Send" key would be in a corner.
> 
> The Enter key is supposed to add a new line into the text box but 
>applications might choose to catch it and still send the message.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/Julow/Unexpected-Keyboard/issues/131#issuecomment-1120245315
> You are receiving this because you authored the thread.
> 
> Message ID: 
***@***.***>

**Comment by dircsem** [comment]
This can be an usage of the space between enter key like that case we were discussing at colemak layout with the delete key.
